source/controllers/SubscetionController.ts

```diff
@@ -24,22 +32,24 @@ subsection.read = async (req, res) => {
 
 interface SubsectionCreateForm {
 	subsection_name: string;
+	lv_subsection_name: string;
 	section_id: string;
 };
 
 interface SubectionEditForm {
 	subsection_name: string;
+	lv_subsection_name: string;
 	section_id: string;
 	subsection_id: string;
 };
 
 function ValidsubsectionCreateForm (obj: any): obj is SubsectionCreateForm {
-	const valid = ('subsection_name' in obj) && typeof obj.subsection_name === 'string' && ('section_id' in obj) && typeof obj.section_id === 'string' && !isNaN(Number(obj.section_id));
+	const valid = ('subsection_name' in obj) && typeof obj.subsection_name === 'string' && ('section_id' in obj) && typeof obj.section_id === 'string' && !isNaN(Number(obj.section_id)) && ('lv_subsection_name' in obj) && typeof obj.lv_subsection_name === 'string';
 	return valid;
 };
 
 function ValidsubsectionUpdateForm (obj: any): obj is SubectionEditForm {
-	const valid = ('subsection_name' in obj) && typeof obj.subsection_name === 'string' && ('section_id' in obj) && typeof obj.section_id === 'string' && ('subsection_id' in obj) && typeof obj.subsection_id === 'string' && !isNaN(Number(obj.section_id)) && !isNaN(Number(obj.subsection_id));
+	const valid = ('subsection_name' in obj) && typeof obj.subsection_name === 'string' && ('section_id' in obj) && typeof obj.section_id === 'string' && ('subsection_id' in obj) && typeof obj.subsection_id === 'string' && !isNaN(Number(obj.section_id)) && !isNaN(Number(obj.subsection_id)) && ('lv_subsection_name' in obj) && typeof obj.lv_subsection_name === 'string';
```

Comment:
long conditionals. If you update it, please at least split it into multiple lines